src/map/entities/petentity.cpp

```diff
@@ -300,6 +300,10 @@ void CPetEntity::OnPetSkillFinished(CPetSkillState& state, action_t& action)
     {
         findFlags |= FINDFLAGS_PET;
     }
+    if ((PSkill->getValidTargets() & TARGET_IGNORE_BATTLEID) == TARGET_IGNORE_BATTLEID)
```

Comment:
If this isn't related to the if-statement above it, please separate them with a newline